requirements.txt

```diff
@@ -20,7 +20,7 @@ Jinja2==2.11.2; python_version >= '3.8'
 libsass==0.17.0
 lxml==3.7.1 ; sys_platform != 'win32' and python_version < '3.7'
 lxml==4.3.2 ; sys_platform != 'win32' and python_version == '3.7'
-lxml==4.6.1 ; sys_platform != 'win32' and python_version > '3.7'
+lxml==4.6.2 ; sys_platform != 'win32' and python_version > '3.7'
```

Comment:
@d-fence Considering that Debian Testing (Bullseye, hard freeze in March 2021) and Ubuntu 21.04 (Hirsute) both come with Python 3.9 and lxml 4.6.2, could we perhaps simplify the complex requirements here and mandate 4.6.2 all the time, in order to get the latest fixes?

The [changelog](https://lxml.de/4.6/changes-4.6.1.html) does not show any major compatibility concerns between 3.7 and 4.6, as we're not using any of the advanced lxml features.

Reply:
Yes, I agree with this simplification in master.